src/os/olcs/imageryprovider.js

```diff
@@ -291,7 +291,7 @@ os.olcs.ImageryProvider.createCreditForSource = function(source) {
   }
 
   return (goog.isDef(imageUrl) || text.length > 0) ?
-         new Cesium.Credit(text, imageUrl, link) : null;
+         new Cesium.Credit({'text': text, 'imageUrl': imageUrl, 'link': link}) : null;
```

Comment:
In the externs, create a `@typedef` (something like `Cesium.CreditOptions`) that defines which properties are available in this options object. Then cast the object here and remove the quoted properties. This allows the compiler to type check the values passed, while preventing them from being minified.